internal/controllers/nodequotaconfig_controller.go

```diff
@@ -121,10 +121,10 @@ func (r *NodeQuotaConfigReconciler) CalculateRootSubnamespaces(ctx context.Conte
 			rootResources = utils.MergeTwoResourceList(secondaryRootSns.Spec.ResourceQuotaSpec.Hard, rootResources)
 		}
 		if err := utils.UpdateRootSubnamespace(ctx, rootResources, rootSubnamespace, logger, r.Client); err != nil {
-			return false, err
+			return false, nil
```

Comment:
Log the error instead of returning, and explain with a `godoc` why this is needed since it's a non-trivial thing to do